features/step_definitions/task_steps.rb

```diff
@@ -0,0 +1,22 @@
+Given(/^a task:$/) do |table|
+  d.given_task table
+  @project = Project.find vertical_table(table)[:project_id]
```

Comment:
So in theory here you are creating a task and giving the `project_id` as part of the params to that task so it is associated with the project, which means if you want `@project` you could do:
`@project = d.given_task(table).project`
Or if you wanted the task for anything you could have done
`@task = d.given_task table`
`@project = @task.project`